.docksal/commands/recipe-apply

```diff
@@ -4,7 +4,7 @@
 
 ## Apply a Drupal Recipe that has been installed using composer.
 ##
-## Usage: fin recipe-apply [name]
+## Usage: fin recipe-apply /path/to/[recipe_name]
```

Comment:
may I suggest adding:

`## Example: fin recipe-apply ../recipes/recipe-name or fin recipe-apply core/recipes/standard`

If a recipe noob like me runs into this it may take a bit to figure out the right way to do it.

Reply:
Updated.  Great suggestion.